src/components/2dmap/TwoDimensionalMap.tsx

```diff
@@ -261,44 +255,6 @@ export default class TwoDimensionalMap extends React.Component<ITwoDimensionalMa
       }
     }
   }
-
-  /**
-   * Finds the dimensions of a polygon
-   * from 0,0 to the largest x and y
-   * @param polygon the outer polygon
-   * @returns A coordinate object, that contains the length and height of the polygon
-   */
-  public findDimensions(polygon: IPolygon): ICoord {
```

Comment:
Is it on purpose to delete findDimensions()?

Reply:
I see that you have put it in a helper class instead - please explain this choice. :)

Reply:
Now that we itbis connected to the store usinh connect(... map state to props and map dispatch to props) you cant instaciate the class like you did before and also it is very bad practice yo instaciate a react component lile that therefore i moved it and tested it on its own